packages/core/src/actions/send.ts

```diff
@@ -99,23 +99,39 @@ export function send<
           : params.delay;
       }
 
-      let resolvedTarget = isFunction(params.to)
+      const resolvedTarget = isFunction(params.to)
         ? params.to(ctx, _event.data, meta)
         : params.to;
-      resolvedTarget =
-        isString(resolvedTarget) &&
-        resolvedTarget !== SpecialTargets.Parent &&
-        resolvedTarget !== SpecialTargets.Internal &&
-        resolvedTarget.startsWith('#_')
-          ? resolvedTarget.slice(2)
-          : resolvedTarget;
+      let targetActorRef: AnyActorRef | undefined;
+
+      if (typeof resolvedTarget === 'string') {
+        if (resolvedTarget === SpecialTargets.Parent) {
+          targetActorRef = actorContext?.self._parent;
+        } else if (
+          resolvedTarget === SpecialTargets.Internal ||
+          // SCXML compatibility
+          resolvedTarget === '#_internal'
+        ) {
+          targetActorRef = actorContext?.self;
+        } else if (resolvedTarget?.startsWith('#_')) {
+          // SCXML compatibility
+          targetActorRef = state.children[resolvedTarget.slice(2)];
+        } else {
+          targetActorRef = state.children[resolvedTarget];
+          if (!targetActorRef) {
+            throw new Error('no');
```

Comment:
This should come with a meaningful error ;p

Reply:
![CleanShot 2022-09-09 at 08 35 52](https://user-images.githubusercontent.com/1093738/189351202-54cad87b-0f69-4af3-93e8-68f71117052a.png)